README.md

```diff
@@ -67,7 +67,6 @@ Install [swiftenv](https://github.com/kylef/swiftenv/)
 
 ```
 $ cd reminders-cli
-$ swiftenv install
```

Comment:
With `.swift-version` removed the line above generates an error requesting a version. Is removing this line correct?

Reply:
Yea nice catch, this was an outdated hope, really we just expect people to have Xcode setup at this point